src/components/entry/Lightbox.js

```diff
@@ -0,0 +1,150 @@
+  border: 5px solid ${({ theme }) => theme.orange};
+  border-radius: 3px;
+  img {
+    border-radius: 0;
+    border: 0;
+    padding: 0;
+  }
+`
+const NavButtonContainer = styled.div`
+  position: absolute;
+  bottom: 50px;
+  right: 0px;
+`
+
+const ExitButton = styled(ResetButton)`
+  position: absolute;
+  top: 5px;
+  right: 5px;
+  color: ${({ theme }) => theme.secondaryText};
+`
+
+const ImageContainer = styled.div`
+  position: relative;
+`
+
+const NavButton = styled(ResetButton)`
+  color: white;
+  width: 50px;
+  height: 50px;
+  border-radius: 14px;
+  margin-right: 10px;
+  background: rgba(0, 0, 0, 0.65);
+  box-shadow: 0px 4px 4px ${({ theme }) => theme.shadow};
+`
+
+const Lightbox = ({
+  onDismiss,
+  review,
+  currReviewIndex,
+  reviewSubImgIndex,
+  setCurrReviewIndex,
+  setReviewSubImgIndex,
+  reviewImages,
+}) => {
+  const incrementReviewImage = () => {
+    reviewSubImgIndex + 1 < reviewImages[currReviewIndex].length
+      ? setReviewSubImgIndex(reviewSubImgIndex + 1)
+      : currReviewIndex + 1 < reviewImages.length
+      ? (setCurrReviewIndex(currReviewIndex + 1), setReviewSubImgIndex(0))
+      : (setCurrReviewIndex(0), setReviewSubImgIndex(0))
+  }
+  // TODO: Fix errors within this function (to test, spam back button)
+  //   Will convert to ternary when working
+  const decrementReviewImage = () => {
+    if (reviewSubImgIndex - 1 < 0) {
+      if (currReviewIndex - 1 < 0) {
+        setCurrReviewIndex(reviewImages.length - 1)
+        setReviewSubImgIndex(reviewImages[currReviewIndex].length)
+      } else {
+        console.log('before', currReviewIndex)
+        setCurrReviewIndex(currReviewIndex - 1)
+        // For some reason, the curr review index is the same in both cases
+        // This occurs when moving from the first landscape pic back to the foraging pic
+        console.log('after', currReviewIndex)
+        setReviewSubImgIndex(reviewImages[currReviewIndex].length)
+      }
+    } else {
+      setReviewSubImgIndex(reviewSubImgIndex - 1)
+    }
+  }
+  return (
+    <StyledDialog onDismiss={onDismiss}>
+      <ImageContainer>
+        <StyledReviewImage
+          src={reviewImages[currReviewIndex][reviewSubImgIndex].medium}
+        />
+        <NavButtonContainer>
+          <NavButton onClick={decrementReviewImage}>
+            <LeftArrowAlt size={30} />
+          </NavButton>
+          <NavButton onClick={incrementReviewImage}>
+            <RightArrowAlt size={30} />
+          </NavButton>
+        </NavButtonContainer>
+      </ImageContainer>
+      <ReviewContainer>
+        <ExitButton onClick={onDismiss}>
+          <X size={30} />
+        </ExitButton>
+        <Review review={review[currReviewIndex]} includePreview={false} />
+
+        <ThumbnailImages>
+          {reviewImages[currReviewIndex].map((photo) =>
+            photo.thumb ===
+            reviewImages[currReviewIndex][reviewSubImgIndex].thumb ? (
+              <SelectedImagePreview $small key={photo.thumb}>
```

Comment:
Maybe we could use a wrapped version of `SelectedImagePreview` for both selected and unselected instances and have a prop that dictates if an instance is currently selected.